packages/suite/src/actions/wallet/send/sendFormThunks.ts

```diff
@@ -80,44 +90,144 @@ export const importSendFormRequestThunk = createThunk(
     (_, { dispatch }) => dispatch(modalActions.openDeferredModal({ type: 'import-transaction' })),
 );
 
+const updateRbfLabelsThunk = createThunk(
+    `${MODULE_PREFIX}/updateReplacedTransactionThunk`,
+    (
+        {
+            labelsToBeEdited,
+            precomposedTx,
+            txid,
+        }: {
+            labelsToBeEdited: RbfLabelsToBeUpdated;
+            precomposedTx: PrecomposedTransactionFinal;
+            txid: string;
+        },
+        { dispatch, extra },
+    ) => {
+        const {
+            thunks: { moveLabelsForRbfAction },
+        } = extra;
+
+        if (labelsToBeEdited !== undefined) {
```

Comment:
can labelsToBeEdited be undefined? It looks like no. If so, can you, please, update type?

Reply:
fixed in e3583e3c18